### PR TITLE
Localize all error messages

### PR DESCRIPTION
**Pull request by jakubmiarka**
This makes it easier to see what content gets used

🚚 📦 Special delivery for @christopherthomasdesign

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -80,6 +82,14 @@ en:
...
     upload: Upload your certificate
     previous: Previous
+    errors:
+      invalid: is not a valid x509 certificate
+      expired: has expired
+      expires_soon: expires too soon
+      valid_too_long: valid for too long
+      not_rsa: is not RSA
+      small_key: key size is less than 2048 bits
+      not_signing: signing certificate required
```
Comment (alan-gds) [inline config/locales/en.yml]: Should this be `no_signing`?

Comment (jakubmiarka) [reply config/locales/en.yml]: The context is someone is trying to disable an encryption cert, though it's only possible to disable signing ones. So saying it's not signing...

Comment (alan-gds) [reply config/locales/en.yml]: 👍

**Comment by jakubmiarka** [comment]
@alan-gds I fixed the missing error on the aggregate event

**Pull request merged**